ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -53,19 +53,119 @@ interface Management1
    vrf MGMT
```

Comment:
below commands are not supported on EOS

ip nat source ingress static 3.0.0.8 4.0.0.8
ip nat destination egress static 239.0.0.1 239.0.0.2
permit response traffic nat
permit ip any any nexthop-group NH_TEST

Reply:
> permit response traffic nat
this is supported on newer versions at least on CloudEOS please make sure to use latest versions and to check across different platforms

```
ip access-list BL
   permit response traffic nat
```

> permit ip any any nexthop-group NH_TEST

not supported on CloudEOS but still supported in CLI on my version

```
site2-wan1(config-acl-BL)#permit ip any any n
  nexthop-group  not supported on this hardware platform
```

Reply:
With respect to cvp change control i see below errors on 4.32.2F-38195967.4322F (engineering build)
```
41 Configuration Errors
AVD_s1-leaf1, line: 0 - > ip nat translation address selection any % Unavailable command (not supported on this hardware platform) (at token 5: 'any')
AVD_s1-leaf1, line: 0 - > ip nat profile NAT-PROFILE-NO-VRF-1 % Unavailable command (not supported on this hardware platform) (at token 2: 'profile')
AVD_s1-leaf1, line: 0 - > ip nat profile NAT-PROFILE-NO-VRF-2 % Unavailable command (not supported on this hardware platform) (at token 2: 'profile')
AVD_s1-leaf1, line: 0 - >    ip nat destination static 1.0.0.1 2.0.0.1 % Unavailable command (not supported on this hardware platform) (at token 2: 'destination')
AVD_s1-leaf1, line: 0 - >    ip nat destination static 1.0.0.2 22 2.0.0.2 % Unavailable command (not supported on this hardware platform) (at token 2: 'destination')
AVD_s1-leaf1, line: 0 - >    ip nat destination static 1.0.0.2 23 2.0.0.3 23 % Unavailable command (not supported on this hardware platform) (at token 2: 'destination')
AVD_s1-leaf1, line: 0 - >    ip nat destination static 1.0.0.4 22 2.0.0.4 23 protocol udp % Unavailable command (not supported on this hardware platform) (at token 2: 'destination')
AVD_s1-leaf1, line: 0 - >    ip nat destination static 1.0.0.7 access-list ACL21 2.0.0.7 % Unavailable command (not supported on this hardware platform) (at token 2: 'destination')
AVD_s1-leaf1, line: 0 - >    ip nat source static 3.0.0.1 4.0.0.1 % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat source static 3.0.0.2 22 4.0.0.2 % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat source static 3.0.0.3 22 4.0.0.3 23 % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat source static 3.0.0.4 22 4.0.0.4 23 protocol udp % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat source static 3.0.0.7 access-list ACL21 4.0.0.7 % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat source ingress static 3.0.0.8 4.0.0.8 % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat destination egress static 239.0.0.1 239.0.0.2 % Unavailable command (not supported on this hardware platform) (at token 2: 'destination')
AVD_s1-leaf1, line: 0 - >    ip nat source static 3.0.0.5 22 4.0.0.5 23 protocol tcp group 1 % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat destination static 1.0.0.5 22 2.0.0.5 23 protocol tcp group 1 % Unavailable command (not supported on this hardware platform) (at token 2: 'destination')
AVD_s1-leaf1, line: 0 - >    ip nat source static 3.0.0.6 22 4.0.0.6 23 protocol tcp group 2 comment Comment Test % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat destination static 1.0.0.6 22 2.0.0.6 23 protocol tcp group 2 comment Comment Test % Unavailable command (not supported on this hardware platform) (at token 2: 'destination')
AVD_s1-leaf1, line: 0 - >    ip nat destination dynamic access-list ACL1 pool POOL1 % Unavailable command (not supported on this hardware platform) (at token 2: 'destination')
AVD_s1-leaf1, line: 0 - >    ip nat source dynamic access-list ACL11 pool POOL11 % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat source dynamic access-list ACL12 pool POOL11 comment POOL11 shared with ACL11/12 % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat source dynamic access-list ACL13 pool POOL13 priority 10 % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat source dynamic access-list ACL14 pool POOL14 priority 1 comment Priority low end % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat source dynamic access-list ACL15 pool POOL15 priority 4294967295 comment Priority high end % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat source dynamic access-list ACL16 pool POOL16 comment Priority default % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat source dynamic access-list ACL17 overload priority 10 comment Priority_10 % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat source dynamic access-list ACL18 pool POOL18 address-only priority 10 comment Priority_10 % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat source dynamic access-list ACL19 pool POOL19 full-cone priority 10 comment Priority_10 % Unavailable command (not supported on this hardware platform) (at token 2: 'source')
AVD_s1-leaf1, line: 0 - >    ip nat destination dynamic access-list ACL2 pool POOL1 comment POOL1 shared with ACL1/2 % Unavailable command (not supported on this hardware platform) (at token 2: 'destination')
AVD_s1-leaf1, line: 0 - >    ip nat destination dynamic access-list ACL3 pool POOL3 priority 10 % Unavailable command (not supported on this hardware platform) (at token 2: 'destination')
AVD_s1-leaf1, line: 0 - >    ip nat destination dynamic access-list ACL4 pool POOL4 priority 1 comment Priority low end % Unavailable command (not supported on this hardware platform) (at token 2: 'destination')
AVD_s1-leaf1, line: 0 - >    ip nat destination dynamic access-list ACL5 pool POOL5 priority 4294967295 comment Priority high end % Unavailable command (not supported on this hardware platform) (at token 2: 'destination')
AVD_s1-leaf1, line: 0 - >    ip nat destination dynamic access-list ACL6 pool POOL6 comment Priority default % Unavailable command (not supported on this hardware platform) (at token 2: 'destination')
AVD_s1-leaf1, line: 0 - > ip nat profile NAT-PROFILE-TEST-VRF vrf TEST % Unavailable command (not supported on this hardware platform) (at token 2: 'profile')
AVD_s1-leaf1, line: 0 - >    permit response traffic nat % Unavailable command (not supported on this hardware platform) (at token 1: 'response')
AVD_s1-leaf1, line: 0 - >    permit response traffic nat % Unavailable command (not supported on this hardware platform) (at token 1: 'response')
AVD_s1-leaf1, line: 0 - >    permit response traffic nat % Unavailable command (not supported on this hardware platform) (at token 1: 'response')
AVD_s1-leaf1, line: 0 - > ip nat pool port_only_1 port-only % Unavailable command (not supported on this hardware platform) (at token 4: 'port-only')
AVD_s1-leaf1, line: 0 - > ip nat pool port_only_2 port-only % Unavailable command (not supported on this hardware platform) (at token 4: 'port-only')
AVD_s1-leaf1, line: 0 - >    port range 1024 65535 % Invalid input (at token 0: 'port')
```

Reply:
@alexeygorbunov As you said most of command works for me when i use FastCli and push the commands via CLI but face issue via CVP as it simply check the hardware compatibly. As you suggested we can move this config on specific host(hardware).